src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private int capasity;
```

Comment:
```suggestion
    private int capacity;
```
can you avoid using this variable?

Reply:
Можна й не використовувати, і замість неї щоразу визначати теперішню довжину масиву (Я вважав, що взяти значенні зі змінної швидше ніж визначати довжину масиву).